terminal.go

```diff
@@ -358,7 +358,7 @@ func (t *terminal) consumeANSIEscape(buf *bufio.Reader) (result readResult, err
 		r = CharLineEnd
 	case '~':
 		if initial == '[' && data == "3" {
-			r = CharDelete // ???
+			r = MetaDeleteKey // this is the key typically labeled "Delete"
```

Comment:
Move comment to const?

Reply:
This seems OK to me...the escape sequence parsing code in terminal.go is where I typically get confused